build/common.ps1

```
@@ -106,6 +106,9 @@ Function Invoke-BuildStep {
             }
             $completed = $true
         }
+        catch {
+            Error-Log $_
+        }
```

@nkolev92 build exceptions will no longer be silently swallowed!